mitodl/ocw-hugo-themes
# feat: update social links

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/3298
Closes https://github.com/mitodl/hq/issues/3195

### Description (What does it do?)

This PR:

- Adds a LinkedIn link to the homepage's `STAY CONNECTED WITH OCW` card.
- Updates icons for LinkedIn and Twitter.


### Screenshots (if appropriate):

<img width="599" alt="Screenshot 2024-01-11 at 4 44 00 PM" src="https://github.com/mitodl/ocw-hugo-themes/assets/71316217/b2b2548f-7014-4153-9628-14acbc6550f6">

<img width="374" alt="Screenshot 2024-01-11 at 4 44 05 PM" src="https://github.com/mitodl/ocw-hugo-themes/assets/71316217/c8c84360-d4cd-4881-82f4-92ffa4c1d248">




### How can this be tested?

1. Navigate to your local `ocw-hugo-themes` directory.
2. Switch to branch `hussaintaj/3298-new-social-icons`
3. Run
    ```
    yarn start www
    ```
4. Open [http://localhost:3000](http://localhost:3000).
5. Observe the card `STAY CONNECTED WITH OCW` near the bottom.
6. Expect to see a LinkedIn social link.
7. Expect all social links to work correctly.
8. Expect the twitter icon to be updated.
9. Observe the social links in the footer.
10. Expect to see new icons for Twitter and LinkedIn.
11. Expect all social links to work.
